features/user_operations.feature

```diff
@@ -4,47 +4,52 @@ Feature: User Operations
   I want to perform CRUD operations on user records
 
   Scenario: Retrieving the list of users
-    When GET request is send to "/api/users?page=1&itemsPerPage="
+    When GET request is send to "/api/users?page=1&itemsPerPage=10"
     Then the response status code should be 200
     And the response should contain a list of users
 
+  Scenario: Retrieving the list of users with missing pagination value
+    When GET request is send to "/api/users?page=1&itemsPerPage="
+    Then the response status code should be 400
+    And the error message should be "Page and itemsPerPage must be greater than or equal to 1."
+
   Scenario: Retrieving the list of users with wrong params
     When GET request is send to "/api/users?page=1&itemsPerPage=-100"
     Then the response status code should be 400
 
   Scenario: Creating a user
-    Given creating user with email "test@mail.com", initials "name surname", password "passWORD1"
+    Given creating user with email "test@mail.com", initials "namesurname", password "passWORD1"
```

Comment:
Please return the “name surname” instead of “namesurname”, because there is a bug inside initials validator instead of changing the behat tests itself